packages/api/src/beacon/routes/lightclient.ts

```diff
@@ -80,17 +85,40 @@ export function getReqSerializers(): ReqSerializers<Api, ReqTypes> {
     },
 
     getUpdates: {
-      writeReq: (start_period, count) => ({query: {start_period, count}}),
-      parseReq: ({query}) => [query.start_period, query.count],
+      writeReq: (start_period, count, format) => ({
+        query: {start_period, count},
+        headers: {accept: format === "ssz" ? mimeTypeSSZ : ""},
+      }),
+      parseReq: ({query, headers}) => [
+        query.start_period,
+        query.count,
+        headers.accept === mimeTypeSSZ ? "ssz" : "json",
+      ],
       schema: {query: {start_period: Schema.UintRequired, count: Schema.UintRequired}},
     },
 
-    getOptimisticUpdate: reqEmpty,
-    getFinalityUpdate: reqEmpty,
+    getOptimisticUpdate: {
+      writeReq: (format) => ({
+        headers: {accept: format === "ssz" ? mimeTypeSSZ : ""},
+      }),
+      parseReq: ({headers}) => [headers.accept === mimeTypeSSZ ? "ssz" : "json"],
+      schema: {},
```

Comment:
In this scenario, is it correct to have the schema as `{}`? If not what should the definition look like?